src/System.Management.Automation/engine/MshObject.cs

```diff
@@ -1846,8 +1835,8 @@ public virtual void GetObjectData(SerializationInfo info, StreamingContext conte
         /// PSObject to be used while replicating the owner for PSMemberSet
         /// </param>
         /// <returns></returns>
-        internal static object GetNoteSettingValue(PSMemberSet settings, string noteName,
-            object defaultValue, Type expectedType,
+        internal static object? GetNoteSettingValue(PSMemberSet? settings, string noteName,
+            object? defaultValue, Type expectedType,
             bool shouldReplicateInstance, PSObject ownerObject)
```

Comment:
Looking how the method is used we can make it non-nullable:
```suggestion
        internal static object GetNoteSettingValue(PSMemberSet? settings, string noteName,
            object defaultValue, Type expectedType,
            bool shouldReplicateInstance, PSObject ownerObject)
```